pkg/blob/controllerserver.go

```diff
@@ -366,36 +366,12 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	defer d.volumeLocks.Release(volName)
 
 	requestName := "controller_create_volume"
-
-	var srcAzcopyAuthEnv []string
-	var srcSubscriptionID, srcResourceGroupName, srcAccountName, srcContainerName, srcPath, srcAccountSASToken string
 	if volContentSource != nil {
 		switch volContentSource.Type.(type) {
 		case *csi.VolumeContentSource_Snapshot:
-			return nil, status.Errorf(codes.InvalidArgument, "VolumeContentSource Snapshot is not yet implemented")
```

Comment:
this should return error, L372 does not make sense

Reply:
`Type` is an `isVolumeContentSource_Type` value and `volContentSource.Type` is not existed when `volContentSource` is nil, so L372 is needed I think. Return error here